build/lib/eksd_releases.sh

```diff
@@ -38,7 +40,12 @@ function build::eksd_releases::load_release_yaml() {
 function build::eksd_releases::get_release_yaml_url() {
     local -r release_branch=$1
     local -r release_number=$(yq e ".${release_branch}.number" ${REPO_ROOT}/EKSD_LATEST_RELEASES)
-    local -r yaml_url="https://distro.eks.amazonaws.com/kubernetes-${release_branch}/kubernetes-${release_branch}-eks-${release_number}.yaml"
+    local -r dev=$(yq e ".${release_branch}.dev" ${REPO_ROOT}/EKSD_LATEST_RELEASES)
```

Comment:
What are you doing about the bundle pointing to the eks distro prod domain for the various artifacts?

Reply:
We are planning on fixing it from the EKS-D side